### PR TITLE
WIP: Issues 1384

### PR DESCRIPTION
**Pull request by raftaar1191**
Description
This PR is for #1384

How Has This Been Tested?
Tested this with for Donations, Donors, Log Requests.

Types of changes
Bug fix (non-breaking change which fixes an issue)

Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/process-donation.php [hunk 1]
```diff
@@ -230,7 +236,18 @@ function give_process_form_login() {
 	give_log_user_in( $user_data['user_id'], $user_data['user_login'], $user_data['user_pass'] );
 
 	if ( $is_ajax ) {
-		echo 'success';
+		$data['status'] = 'success';
```
Comment (ravinderk) [inline includes/process-donation.php]: @raftaar1191 Can you use WordPress core function https://codex.wordpress.org/Function_Reference/wp_send_json_success

File: includes/process-donation.php [hunk 2]
```diff
@@ -219,7 +220,12 @@ function give_process_form_login() {
 			 *
 			 * @since 1.0
 			 */
-			do_action( 'give_ajax_donation_errors' );
+			ob_start();
```
Comment (ravinderk) [inline includes/process-donation.php]: @raftaar1191 Can we use WordPress core function?
https://codex.wordpress.org/Function_Reference/wp_send_json_error

File: includes/process-donation.php [hunk 3]
```diff
@@ -230,7 +236,18 @@ function give_process_form_login() {
 	give_log_user_in( $user_data['user_id'], $user_data['user_login'], $user_data['user_pass'] );
 
 	if ( $is_ajax ) {
-		echo 'success';
+		$data['status'] = 'success';
+		$data['message'] = Give()->notices->print_frontend_notice(
+			sprintf(
+				/* translators: %s: user first name */
+				esc_html__( 'Welcome %s! You have successfully logged into your account.', 'give' ),
+				$user_data['user_first']
```
Comment (ravinderk) [inline includes/process-donation.php]: @raftaar1191 `user_first` can be empty , so show `user_login` in that situation

File: includes/process-donation.php (hunk 2 above)
Comment (raftaar1191) [inline includes/process-donation.php]: Done

Commit id: https://github.com/raftaar1191/Give/commit/41bf6bf616b28fb9ed89d383a1d975fd4e426e2f

File: includes/process-donation.php (hunk 1 above)
Comment (raftaar1191) [inline includes/process-donation.php]: Done

Commit id: https://github.com/raftaar1191/Give/commit/41bf6bf616b28fb9ed89d383a1d975fd4e426e2f

File: includes/process-donation.php (hunk 3 above)
Comment (raftaar1191) [inline includes/process-donation.php]: Done

Commit id: https://github.com/raftaar1191/Give/commit/41bf6bf616b28fb9ed89d383a1d975fd4e426e2f

File: includes/process-donation.php [hunk 4]
```diff
@@ -230,8 +232,17 @@ function give_process_form_login() {
 	give_log_user_in( $user_data['user_id'], $user_data['user_login'], $user_data['user_pass'] );
 
 	if ( $is_ajax ) {
-		echo 'success';
-		give_die();
+		$message = Give()->notices->print_frontend_notice(
+			sprintf(
+				/* translators: %s: user first name */
+				esc_html__( 'Welcome %s! You have successfully logged into your account.', 'give' ),
+				( isset( $user_data['user_first'] ) && ! empty( $user_data['user_first'] )  ) ? $user_data['user_first'] : $user_data['user_login']
```
Comment (ravinderk) [inline includes/process-donation.php]: @raftaar1191 Do not use `isset`, empty will check for both

**Comment by DevinWalker** [comment]
This works well. Thanks!

File: includes/process-donation.php (hunk 4 above)
Comment (raftaar1191) [inline includes/process-donation.php]: Done

**Pull request merged**